torch/export/unflatten.py

```diff
@@ -1538,11 +1545,13 @@ def _sink_params(
         return module_id_to_inputs_removed
 
     graph = module.graph
-    inputs = list(filter(lambda n: n.op == "placeholder", graph.nodes))  # type: ignore[arg-type, union-attr, var-annotated]
+    assert isinstance(graph, torch.fx.Graph)
```

Comment:
Note that this is the only change in this PR that affects runtime behavior.

Clearly the code below seems to rely on `graph` being a `torch.fx.Graph` instance, so I would expect the assert to be the right thing here.